.eslintrc.json

```diff
@@ -7,6 +7,9 @@
       "mocha": true
     },
     "rules": {
+      "parserOptions": {
+        "sourceType": "module"
+    },
```

Comment:
This is in the wrong place. `parserOptions` should be key in the `eslintrc.json` file @amaechi-chuks

Reply:
Like this

```
{
    ....
    "parserOptions": {...},
    "rules": {...}
}
```

It should not be inside `rules` because it is not linting rule. Do you understand?

Reply:
Noted